js/index-template.js

```diff
@@ -0,0 +1,102 @@
+var template = document.querySelector('#event-template').innerHTML;
+var destination = document.querySelector('.events');
+
+
+// PRODUCTION URL
+// prodUrl requires the latest commit hash for the `json` branch (the merge commit from the last PR)
+// Can be the short or long version.
+var commitHash = "04a6741";
+var prodUrl = "https://cdn.rawgit.com/node-girls/node-girls-website/" + commitHash + "/events.json";
+
+// DEVELOPMENT URL
+// To see if your updated events.json is working, set this variable to the name
+// of the branch you're working from (after you've pushed to GitHub)
+var yourWorkingBranchName = "";
```

Comment:
Minor niggle but I feel that lines 14 and 15 should perhaps be in a comment as they're not actually used in production but I'm not bothered if they are kept in.